CMakeLists.txt

```diff
@@ -141,9 +138,14 @@ if(NOT SDL2_NET_FOUND)
                 WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
         endif()
         set(SDL2_NET_DIR "${CMAKE_CURRENT_BINARY_DIR}/SDL2_net-2.0.1")
-        find_package(SDL2_net)
     endif()
 endif()
+find_package(SDL2_net)
+
+find_package(FluidSynth)
+if(FluidSynth_FOUND)
+    install(DIRECTORY soundfonts/ DESTINATION soundfonts)
```

Comment:
@fabiangreffrath This line packages soundfont on WIN32, I don't know what it does on *NIX.